File: Assignment1/double2ndLast.java
import java.util.*;

public class double2ndLast
{
	public static void main(String[] args) 
	{
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter number: ");
		int n = sc.nextInt();
		int l = n%10;
		int x = ((n%100)/10);
		n = ((((n/10)+x)*10)+l);
		System.out.println(n);

		sc.close();
	}
}

File: Assignment1/double2ndLast.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cout << "Enter number: ";
    cin >> n;
    int l = n % 10;
    int x = ((n % 100) / 10);
    n = ((((n / 10) + x) * 10) + l);
    cout << n << endl;
    return 0;
}